org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/formatting/Formatter2Fragment2.xtend

```diff
@@ -72,11 +73,11 @@ class Formatter2Fragment2 extends AbstractStubGeneratingFragment {
 	}
 
 	protected def doGenerateStubFile() {
-		val xtendFile = doGetXtendStubFile
-		xtendFile?.writeTo(projectConfig.runtime.src)
+		val file = doGetStubFile
+		file?.writeTo(projectConfig.runtime.src)
 	}
 
-	protected def doGetXtendStubFile() {
```

Comment:
api change?

Reply:
Personally I wouldn't consider this to be API.

Reply:
Unlikely that someone wants to override this method ... so probably ok ... I could also restore the new name, but it is then misleading. Would stay with the change.